.golangci.yml

```diff
@@ -112,6 +97,9 @@ linters:
     - interfacebloat
 
 issues:
+  # Only show newly introduced problems.
+  new-from-rev: c5b37702e0bead2408ba72aad792ea4ff9fc930f
```

Comment:
hope this is ok?
(see num of issues given the linter version bump: https://github.com/lightninglabs/pool/actions/runs/13659731510/job/38187830849)

Reply:
hmm is something other than this needed to get it to do the thing? linter step still seems to be including the whole code-base

Reply:
Yeah, probably need to add a couple of new linters to the ignore list.